src/main/scripts/org/reaktivity/specification/nukleus/http2/streams/rfc7540/connection.management/connection.has.two.streams/server/source.rpt

```diff
@@ -0,0 +1,306 @@
+
+property sourceInput ${nuklei.streams("http2", "source")}
+property sourceHandshakeEst ${nuklei.streams("source", "http2#source")}
+property sourceOutputEst ${nuklei.streams("source", "http2#target")}
+
+property newSourceInputId ${nuklei:newStreamId()}
+property newSourceCorrelationId ${nuklei:newCorrelationId()}
+
+#
+#
+# writing to source-->nukleus (http2/source)
+#
+#
+connect await ROUTED_INPUT
+        agrona://stream/bidirectional
+        option reader ${agrona:oneToOneReader(sourceInput.throttle)}
+        option writer ${agrona:oneToOneWriter(sourceInput.buffer)}
+connected
+
+# begin frame
+write [0x00 0x00 0x00 0x01]
+write ${newSourceInputId}
+write ${newSourceInputRef}
+write ${newSourceCorrelationId}
+write flush
+
+# window frame
+read [0x40 0x00 0x00 0x02]
+read ${newSourceInputId}
+read [0..4]
+
+# data frame (http2 client connection preface)
+write [0x00 0x00 0x00 0x02]
+write ${newSourceInputId}
+write [0x00 0x18]
+write "PRI * HTTP/2.0\r\n"
+      "\r\n"
+      "SM\r\n"
+      "\r\n"
+write flush
+
+# window frame
+read [0x40 0x00 0x00 0x02]
+read ${newSourceInputId}
+read [0..4]
+
+# data frame (http2 SETTINGS frame)
+write [0x00 0x00 0x00 0x02]
+write ${newSourceInputId}
+write [0x00 0x0f]
+write [0x00 0x00 0x06]                   # length = 6
+      [0x04]                             # HTTP2 SETTINGS frame
+      [0x00]                             # flags = 0x00
+      [0x00 0x00 0x00 0x00]              # stream_id = 0
+      [0x00 0x04 0x00 0x00 0xff 0xff]    # SETTINGS_INITIAL_WINDOW_SIZE(0x04) = 65535
+write flush
+
+# window frame
+read [0x40 0x00 0x00 0x02]
+read ${newSourceInputId}
+read [0..4]
+
+# data frame (http2 SETTINGS frame)
+write [0x00 0x00 0x00 0x02]
+write ${newSourceInputId}
+write [0x00 0x09]
+write [0x00 0x00 0x00]          # length = 0
+      [0x04]                    # HTTP2 SETTINGS frame
+      [0x01]                    # ACK
+      [0x00 0x00 0x00 0x00]     # stream_id = 0
+write flush
+
+# window frame
+read [0x40 0x00 0x00 0x02]
+read ${newSourceInputId}
+read [0..4]
+
+# data frame - HTTP2 HEADERS stream_id=1
+write [0x00 0x00 0x00 0x02]
+write ${newSourceInputId}
+write [0x00 0x18]
+write [0x00 0x00 0x0f]                                                  # length = 15
+      [0x01]                                                            # HEADERS frame
+      [0x05]                                                            # END_STREAM | END_HEADERS
+      [0x00 0x00 0x00 0x01]                                             # stream_id = 1
+      [0x83]                                                            # :method: POST
+      [0x86]                                                            # :scheme: http
+      [0x84]                                                            # :path: /
+      [0x41] [0x8a 0xa0 0xe4 0x1d 0x13 0x9d 0x09 0xb8 0xf0 0x1e 0x07]   # :authority: localhost:8080
```

Comment:
Can we make it in the clear instead for readability please?
The Huffman encoding tests can be done once separately, and the other tests can all remain in the clear.